folium/utilities.py

```diff
@@ -198,6 +198,17 @@ def png_pack(png_tag, data):
         png_pack(b'IEND', b'')])
 
 
+def get_parent_map(element):
+    """Return the first `Map` object found in the line of parents of the element."""
+    if not hasattr(element, '_parent'):
+        raise ValueError("This object does not have a parent folium Map.")
+    parent = element._parent
+    if isinstance(parent, folium.Map):
```

Comment:
No problem, this function can likely be removed. If not, we'll have to think about how to solve the circular import.

Reply:
Yeah no longer necessary in this PR. I'll commit a change removing it. I think it could be a useful feature for other layers though, with many layers of parent/child chaining. Maybe best to include directly in `folium.py` instead of `branca.utilities.py`? It can be addressed in a separate PR.